RetrospectiveExtension.Frontend/utilities/external/telemetryClient.ts

```diff
@@ -6,11 +6,28 @@ const appInsights = new ApplicationInsights({
   config: {
     instrumentationKey: environment.AppInsightsInstrumentKey,
     extensions: [reactPlugin],
+    loggingLevelConsole:2,
+    loggingLevelTelemetry:2,
     extensionConfig: {
       [reactPlugin.identifier]: { history: {} }
     }
   }
 });
 appInsights.loadAppInsights();
 
+
+// The new log error function logs the data to console as usual,
+// but also sends error messages to Application insights as well.
+var updatedConsoleError=(function(oldError){
```

Comment:
nit: naming oldError to incomingError

Reply:
`oldError` here is a itself a function and not the incming error message. It is percisely the old `console.error` function. I renamed the function to `oldErrorFunction` to make it more explicit